examples/quickstart/jupyter-notebooks/Python_API_Tutorial.ipynb

```diff
@@ -0,0 +1,1262 @@
+      "    SCHEMA_SCHEMA = 'INFORMATION_SCHEMA'\n",
+      "    SQL_ARRAY = 'array'\n",
+      "    SQL_ARRAY_TYPE = 'ARRAY'\n",
+      "    SQL_ARRAY_WITH_TRAILER = 'arrayWithTrailer'\n",
+      "    SQL_BIGINT_TYPE = 'BIGINT'\n",
+      "    SQL_CSV = 'csv'\n",
+      "    SQL_DOUBLE_TYPE = 'DOUBLE'\n",
+      "    SQL_FLOAT_TYPE = 'FLOAT'\n",
+      "    SQL_OBJECT = 'object'\n",
+      "    SQL_TIMESTAMP_TYPE = 'TIMESTAMP'\n",
+      "    SQL_VARCHAR_TYPE = 'VARCHAR'\n",
+      "    SUCCESS_STATE = 'SUCCESS'\n",
+      "    SYS_SCHEMA = 'sys'\n",
+      "    TABLES_TABLE = 'INFORMATION_SCHEMA.TABLES'\n",
+      "\n",
+      "FILE\n",
+      "    /Users/paul/git/druid/examples/quickstart/jupyter-notebooks/druidapi/consts.py\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "help(consts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b661b29f",
+   "metadata": {},
+   "source": [
+    "Using the constants avoids typos:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "id": "3393af62",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div class=\"druid\"><table>\n",
+       "<tr><th>TableName</th></tr>\n",
+       "<tr><td>segments</td></tr>\n",
+       "<tr><td>server_segments</td></tr>\n",
+       "<tr><td>servers</td></tr>\n",
+       "<tr><td>supervisors</td></tr>\n",
+       "<tr><td>tasks</td></tr>\n",
+       "</table></div>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sql_client.show_tables(consts.SYS_SCHEMA)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e789ca7",
+   "metadata": {},
+   "source": [
+    "## Tracing\n",
+    "\n",
+    "It is often handy to see what the Druid API is doing: what messages it sends to Druid. You may need to debug some function that isn't working as expected. Or, perhaps you want to see what is sent to Druid so you can replicate it in your own code. Either way, just turn on tracing:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "id": "ac68b60e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "druid.trace(True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7b9dc7e3",
+   "metadata": {},
+   "source": [
+    "Then, each call to Druid prints what it sends:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "id": "72c955c0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "            SELECT TABLE_NAME AS TableName\n",
+      "            FROM INFORMATION_SCHEMA.TABLES\n",
+      "            WHERE TABLE_SCHEMA = 'druid'\n",
+      "            ORDER BY TABLE_NAME\n",
+      "            \n",
+      "POST: http://localhost:8888/druid/v2/sql\n",
```

Comment:
I think we should normally clear the outputs before we check in, but this is rendering a little strangely for me. Not sure if we can add an additional newline to the library to give some space between the trace
<img width="685" alt="Screen Shot 2023-02-21 at 3 23 43 PM" src="https://user-images.githubusercontent.com/38529548/220481641-a159bdbf-3411-484c-b880-b7dab325de80.png">

Reply:
You raise to questions. First, clearing the notebook. I agree that we should normally do that. Since this one is a quick intro to the API, I experimented with leaving the output as part of the reference so the user doesn't have to step through the notebook to see what the various methods do. I can be persuaded otherwise, especially if we eventually create real documentation for the library. (There are tools to generate documentation from the Python code itself.)

On the rendering, yes, it is odd. The reason is that the above prints a string that has embedded newlines and spaces because the SQL statement itself has these. It looks a bit odd, but it works.